server/src/test/proto/spine/test/command/commands.proto

```diff
@@ -43,8 +43,28 @@ message CmdStartProject {
     ProjectId project_id = 1;
 }
 
+message CmdRemoveTask {
+    ProjectId project_id = 1;
+    Task task = 2;
+}
+
 // A command to create a project for duplicate command tests.
 //
 message DCmdCreateProject {
     ProjectId project_id = 1;
 }
+
+// A command to create a project to test the command posting order.
+//
+message FCmdCreateProject {
+
+    ProjectId id = 1;
+
+    string name = 2;
+}
+
+message FCmdStartProject {
```

Comment:
If `F` stands for `First` I would say so. And its `TestEnv` factory methods starting with `f` are also unreadable. Let's make their naming more user-friendly.

And the Message description is needed.